### ISSUE TITLE
Request API endpoint to query single instance of /ping 

### ISSUE THREAD

**Issue by saali2016** [ansible/awx#6406]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
currently when we query /api/v2/ping/ we get the health-check for all the instances in the cluster we need is way to query he API and only display the information of that instance.
e.g
~~~
curl -s 'https://<myserver1>/api/v2/ping/?active_node_only=true'

{
      "version": "3.6.0",
      "capacity": 19,
      "heartbeat": "2020-03-24T21:13:15.271164Z",
      "uuid": "xxx-xxxx-xxx-xxxx-xxxx",
      "node": "tower1"
    },
~~~

**Comment by ryanpetrello** [comment]
In a load-balanced environment, or an environment behind a proxy, there's not really any way to *know* this information.

**Comment by jakemcdermott** [comment]
yep - you'll need to do this sort of filtering and matching in the client you're using to query the api.

**Comment by cdggcali** [comment]
> yep - you'll need to do this sort of filtering and matching in the client you're using to query the api.

@jakemcdermott there are products like Akamai's Global Traffic Management that are only capable of parsing XML files. Since XML is not supported and there are no plans to add additional formats (https://github.com/ansible/awx/issues/6274), the only way to use Akamai's GTM load balancer is to read the `/api/v2/ping/` as text file.

With the current output, we would not be able to differentiate which is the capacity of the node we are querying, this is why we opened a ticket with Red Hat and they opened RFE.

Is it possible to re-open this RFE?
